### PR TITLE
Add some missing and hidden versions

### PR DESCRIPTION
**Pull request by rtm516**
I haven't tested the `x86_64` version codes, as I don't have a valid device ID for those. There is also version 1.12.0.27 but that has been excluded as it doesn't boot.

### CODE REVIEW

File: versions.json [hunk 1]
```diff
@@ -2001,6 +2025,36 @@
         },
         "beta": true
     },
+    {
+        "version_name": "1.16.0.52",
+        "codes": {
+            "armeabi-v7a": 941160052,
+            "x86": 942160052,
+            "arm64-v8a": 943160052,
+            "x86_64": 944160052
```
Comment (ChristopherHX) [inline versions.json:2034]: I was pretty shure 1.16.0.55 was the first x86_64 build.
But today I only was able to download beta 1.16.0.65 arm64 / x86_64 with the aurora app, need to check the launcher again.

**Comment by ChristopherHX** [comment]
All old arm32/x86 betas are fine.
The 64bit builds needs to be checked, maybe needs to be removed from this PR., all failed for me to download with the aurora app.

**Comment by ChristopherHX** [comment]
@rtm516 I checked **1.16.0.52 1.16.0.53, 1.16.0.54** and they won't download at all via the gplayapi,
please remove the 64bit (arm64 and x86_64) beta entries below 1.16.0.55, you have added.

I will merge your Pull Request as soon as you have removed the mentioned 64bit versions.

**Comment by rtm516** [comment]
> @rtm516 I checked **1.16.0.52 1.16.0.53, 1.16.0.54**, they failed to download via the gplayapi,
> please remove the 64bit (arm64 and x86_64) beta entries below 1.16.0.55 from your PR.
> 
> I will merge your Pull Request as soon as you have removed the mentioned 64bit versions.

Thanks for testing them I must have not checked properly. Should be all sorted now.

**Pull request merged**